server/src/main/java/org/elasticsearch/index/engine/CombinedDeletionPolicy.java

```diff
@@ -227,7 +243,8 @@ protected IndexCommit wrapCommit(IndexCommit indexCommit) {
      * @return true if the acquired commit can be clean up.
      */
     synchronized boolean releaseCommit(final IndexCommit acquiredCommit) {
-        final IndexCommit releasingCommit = ((SnapshotIndexCommit) acquiredCommit).getIndexCommit();
+        SnapshotIndexCommit snapshotIndexCommit = (SnapshotIndexCommit) acquiredCommit;
```

Comment:
Nit: consistency with the local code style
```suggestion
        final SnapshotIndexCommit snapshotIndexCommit = (SnapshotIndexCommit) acquiredCommit;
```